tests/test_core.py

```diff
@@ -7474,10 +7471,12 @@ def do_test_coroutine(self, additional_settings):
     self.do_run(src, '*leaf-0-100-1-101-1-102-2-103-3-104-5-105-8-106-13-107-21-108-34-109-*')
 
   @no_wasm_backend('ASYNCIFY coroutines are not yet supported in the LLVM wasm backend')
+  @no_fastcomp('ASYNCIFY has been removed from fastcomp')
```

Comment:
Is there a simpler way to skip a test entirely?

Reply:
Not atm. Once we remove fastcomp this will look less silly...